electron/lib/room-util.js

```diff
@@ -1,9 +1,11 @@
 const fs = require('fs')
 const util = require('util')
 
+// No need to promisify: https://stackoverflow.com/a/52094177/1957884
+// - "Starting in Node v10 you can use await fs"
```

Comment:
Why not to use await fs... instead of a comment?

Reply:
It was out if scope and wasn't sure about test coverage but wanted to add a FIXME note (without fixme prefix 😹) to remember that.